internal/component/redis/metadata.go

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I do not want put redis into `internal`.

Reply:
ok
Actually I just want to move these code from  redis state store package to some other package,so that redis lock store can reuse these code.
Can you offer some advice, which package should I move it to?

Reply:
May be `pkg/redis`.

Reply:
@daixiang0 Hi, I found that before this PR, there was already a `internal/component/redis` directory, see https://github.com/dapr/components-contrib/tree/master/internal/component/redis
Both redis pubsub component and redis state component import this package:
![image](https://user-images.githubusercontent.com/26001097/164119053-969a975b-00ac-4186-b744-2f50aa238159.png)
And that's why I put the metadata into this directory too....
Should I move all the code already in `internal/component/redis` to `pkg/redis` ?

Reply:
Just keep it, thanks.